metapool/amplipool.py

```diff
@@ -0,0 +1,63 @@
+import os
+import pandas as pd
+
+from metapool.plate import _decompress_well, _plate_position
+
+
+def assign_emp_index(plate_df, metadata):
+    """Assign an EMP index to wells based on their compressed position
+
+    Parameters
+    ----------
+    plate_df: pd.DataFrame
+        Object with a Well column, and other metadata variables (usually with
+        384 rows).
+    metadata: pd.DataFrame
+        Object with all the plate metadata (usually with 1-4 rows).
+
+    Returns
+    -------
+    pd.DataFrame
+        A table resulting from joining the compressed plate, the plate
+        metadata, and the EMP indices.
+    """
+    plate_df = plate_df.copy()
+
+    # dataframe of wells organized by plate with a barcode per well
+    emp_indices = _load_emp_indices()
+
+    # the Well column exists elsewhere already so we rename it to avoid
+    # duplication in the merged table
+    emp_indices['EMP Primer Plate Well'] = emp_indices['Well'].copy()
+    emp_indices.drop(['Well'], axis=1, inplace=True)
```

Comment:
If I'm following this correctly, what about using rename?

Reply:
Great suggestion, fixed!